pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -101,18 +114,29 @@ func (e *Executor) checkApproval(ctx context.Context) (string, bool) {
 		return "", false
 	}
 
+	as, ok := e.validateApproverNum(ctx, approveCmd.Commander, num)
+	if len(as) > 0 {
+		if err := approveCmd.Report(ctx, model.CommandStatus_COMMAND_SUCCEEDED, nil, nil); err != nil {
+			e.Logger.Error("failed to report handled command", zap.Error(err))
+		}
+	} else {
+		if err := approveCmd.Report(ctx, model.CommandStatus_COMMAND_FAILED, nil, nil); err != nil {
+			e.Logger.Error("failed to report command status", zap.Error(err))
+		}
+	}
```

Comment:
I'm guessing it should handle the command as `SUCCEEDED` even if it's the command as the same user, but what do you think.

Also, if you check if it's valid by checking `as`, `if as == ""` is enough because it's string.

Reply:
To my understanding, `CommandStatus_COMMAND_FAILED` is for cases where it couldn't handle command for some reason kind of like unexpected errors.

Reply:
Oh, I see.
My concern was that it may confuse the user because the popup will show even though it is failed.

Reply:
However, I get your point. I will change it, too.